lib/binaries/Runtime4Docker.js

```diff
@@ -24,12 +24,12 @@ commander.version(pkg.version)
   .option('--delay <seconds>', 'delay start of configuration file by <seconds>', 0)
   .option('--web [port]', 'launch process web api on [port] (default to 9615)')
   .option('--only <application-name>', 'only act on one application of configuration')
-  .option('--auto-exit', 'exit if all processes are errored/stopped or 0 apps launched', true)
+  .option('--no-auto-exit', 'do not exit if all processes are errored/stopped or 0 apps launched')
   .option('--env [name]', 'inject env_[name] env variables in process config file')
   .option('--watch', 'watch and restart application on file change')
   .option('--error <path>', 'error log file destination (default disabled)', '/dev/null')
   .option('--output <path>', 'output log file destination (default disabled)', '/dev/null')
-  .usage('pm2-docker app.js');
```

Comment:
The help usage was `Usage: pm2-docker pm2-docker app.js`, so this change removes the duplicate `pm2-docker`.